test/sanity/ignore.txt

```diff
@@ -269,6 +269,8 @@ test/integration/targets/collections_relative_imports/collection_root/ansible_co
 test/integration/targets/expect/files/test_command.py future-import-boilerplate
 test/integration/targets/expect/files/test_command.py metaclass-boilerplate
 test/integration/targets/gathering_facts/library/bogus_facts shebang
+test/integration/targets/gathering_facts/library/facts_one shebang
+test/integration/targets/gathering_facts/library/facts_two shebang
```

Comment:
Is there a reason why the scripts cannot use `#!/usr/bin/env bash` instead of adding ignores?

Reply:
To answer my own question, it's because these are detected as modules and the sanity test does not yet allow shell shebangs for modules.

Reply:
There's an open issue relating to this: https://github.com/ansible/ansible/issues/68685

Reply:
also, using 'env' breaks ansible_bash_intepreter usage